.travis.yml

```diff
@@ -67,20 +66,17 @@ aliases:
     ext/bin/check-spec-env "$PDB_TEST"
     ext/bin/run-rspec-tests "$puppet_ref"
 
-  - &run-docker-tests |
-    set -ex
```

Comment:
double-check whether we need `set -e`

Reply:
Double checked.. its needed... and fixed.

Thanks!